geoh5py/data/boolean_data.py

```diff
@@ -29,12 +28,6 @@ class BooleanData(ReferencedData):
     Data class for logical (bool) values.
     """
 
```

Comment:
BOOLEAN_VALUE_MAP got moved to `reference_value_map` module. The `validate_keys` of the class checks that it can only be 0:False, 0: True

Reply:
Ok got it. 
But shouldn't the fixed reference_value_map shouldn't be set in the BooleanData rather than in another utility function somewhere else?Just a comment, I'm ok if you rather doing it somewhere else.

Reply:
I don't think so, since BooleanData doesn't know about value_map - only its data_type